std_msgs/Header header

ObjectDescription description

geometry_msgs/Vector3 pedestrian_velocity